oxd-server/src/main/java/org/xdi/oxd/server/op/GetAccessTokenByRefreshTokenOperation.java

```diff
@@ -75,4 +78,10 @@ private String scopeAsString(GetAccessTokenByRefreshTokenParams params) throws U
         }
         return Utils.joinAndUrlEncode(scope);
     }
+    private void validate(GetAccessTokenByRefreshTokenParams params) {
```

Comment:
there should be new line before new method. Try to use automatic IDE formatting.